docs/docs/overview/performance/application.md

```diff
@@ -0,0 +1,126 @@
+# Application benchmarks
+
+## HashiCorp Vault
+
+[HashiCorp Vault](https://www.vaultproject.io/) is a distributed secrets management software that can be deployed to Kubernetes.
+HashiCorp maintains a benchmarking tool for vault, [vault-benchmark](https://github.com/hashicorp/vault-benchmark/).
+Vault-benchmark generates load on a Vault deployment and measures response times.
+
+This article describes the results from running vault-benchmark on Constellation, AKS, and GKE.
+The setup for producing the data discussed in this article can be found in the [vault-benchmarks](https://github.com/edgelesssys/vault-benchmarks) repository.
+
+The Vault API used during benchmarking is the [transits secret engine](https://developer.hashicorp.com/vault/docs/secrets/transit).
+This allows services to send data to vault for encryption, decryption, signing, and verification.
+
+## Results
+
+On each run vault-benchmark sends requests and measures the latency for each request.
+The measured latencies are aggregated through various statistical features.
+After running the benchmark n times the arithmetic mean over a subset of the reported statistics is calculated.
+The selected features are arithmetic mean, 99th percentile, minimum, and maximum.
+
+Arithmetic mean gives a general sense of the latency on each target.
+The 99th percentile shows performance in (most likely) erroneous states.
+Minimum and maximum mark the range within which latency varies each run.
+
+The benchmark was configured with 1300 workers and 10 seconds per run.
+1300 workers was chosen as requests started to fail with more workers.
+10 seconds was chosen as the latency didn't change with longer runs.
+All results are based on 100 runs.
+
+### 5 Replicas
+
+The following data was generated while running 5 replicas, i.e. 1 primary, 4 standby nodes.
+All numbers are in seconds, if not indicated otherwise.
```

Comment:
```suggestion
The following data was generated while running 5 replicas, i.e. 1 primary, 4 standby nodes.
All numbers are in seconds if not indicated otherwise.
```